scripts/performance/perf_processes.py

```diff
@@ -608,14 +661,260 @@ class RGGetRevocRegDelta(RGGetEntryRevoc):
+                                                            self._submitter_did, nym_req)
+            ensure_is_reply(nym_resp)
+
+            trustee_dids.append(tr_did)
+
+        return trustee_dids
+
+    async def __create_payment_addresses(self):
+        for i in range(self._payment_addrs_count):
+            self._payment_addresses.append(
+                await payment.create_payment_address(self._wallet_handle, self._payment_method, '{}'))
+
+    async def __mint_sources(self, payment_addresses, trustees_dids):
+        outputs = []
+        for payment_address in payment_addresses:
+            outputs.append({"recipient": payment_address, "amount": random.randint(1, RGBasePayment.AMOUNT_LIMIT)})
+
+        mint_req, _ = await payment.build_mint_req(self._wallet_handle,
+                                                   self._submitter_did,
+                                                   json.dumps(outputs),
+                                                   None)
+
+        for trustee_did in trustees_dids:
+            mint_req = await ledger.multi_sign_request(self._wallet_handle, trustee_did, mint_req)
+
+        mint_resp = await ledger.submit_request(self._pool_handle, mint_req)
+        ensure_is_reply(mint_resp)
+
+    async def _get_payment_sources(self, payment_address):
+        get_payment_sources_req, _ = \
+            await payment.build_get_payment_sources_request(self._wallet_handle,
+                                                            self._submitter_did,
+                                                            payment_address)
+        get_payment_sources_resp = \
+            await ledger.sign_and_submit_request(self._pool_handle,
+                                                 self._wallet_handle,
+                                                 self._submitter_did,
+                                                 get_payment_sources_req)
+        ensure_is_reply(get_payment_sources_resp)
+
+        source_infos_json = \
+            await payment.parse_get_payment_sources_response(self._payment_method,
+                                                             get_payment_sources_resp)
+        source_infos = json.loads(source_infos_json)
+        payment_sources = []
+        for source_info in source_infos:
+            payment_sources.append((source_info["source"], source_info["amount"]))
+        return payment_sources
+
+
+class RGGetPaymentSources(RGBasePayment):
+    def _gen_req_data(self):
+        return random.choice(self._payment_addresses)
+
+    async def _gen_req(self, submit_did, req_data):
+        payment_address = req_data
+        req, _ = await payment.build_get_payment_sources_request(self._wallet_handle,
+                                                                 self._submitter_did,
+                                                                 payment_address)
+        return req
+
+
+class RGPayment(RGBasePayment):
```

Comment:
Should we add this to `create_req_generator`?

Reply:
Added missed entries to the map.